app/src/main/java/org/onionshare/android/server/WebserverManager.kt

```diff
@@ -58,17 +57,18 @@ class WebserverManager @Inject constructor() {
                 loader(ClasspathLoader().apply { prefix = "assets/templates" })
             }
             installStatusPages(staticPathMap)
+            // this method will not return until the continuation in that listener gets resumed
```

Comment:
Does this mean that start() won't return until state.value is set by one of the handlers that are subscribed in addListener()? Or does it mean than addListener() won't return until that happens? Or something else?

Reply:
Sorry that's a stale comment. Originally, before implementing auto-stop of the webserver, I passed a continuation to the listener, so the start call would suspend/block until the server actually started, but now I handle this different with the `MutableState`.

Reply:
Thank God for that, I thought some deep continuation magic must be happening here. :-)